### ISSUE TITLE
Adaptive css theme

### ISSUE THREAD

**Issue by oldcastlehq** [PhilippeMarcMeyer/vanillaSelectBox#74]
Hi,

I really like this script, it's super fast and get the job done. My only issue is about the css theme. Could it be adaptive? For example, I'm building a CRUD and using [Tabler](https://tabler.io/) for that. I'm having a hard time to get vanillaSelectBox into the Tabler style. Is there a way around?

Cheers.

**Comment by tleifj** [comment]
I agree. Additionally, can you not have the border and background have an important inline style? It makes it impossible to override those styles.
